docs/annotations.md

```diff
@@ -1,12 +1,17 @@
-# Table of Contents
-- [Introduction](#introduction)
-- [Backend path prefix](#backend-path-prefix)
-- [SSL Redirect](#ssl-redirect)
-
 ## Introduction
 Kubernetes Ingress specification allows for annotations. We use annotations to expose Application Gateway specific features that can't be exposed using the ingress specification. It is important to note that annotations defined on an ingress resource are applied to all HTTP setting, backend pools and listeners defined within a given ingress resource.
 
-## Backend path prefix
+### List of supported annotations
+| Annotation Key | Value Type | Default Value |
+| -- | -- | -- |
+| [appgw.ingress.kubernetes.io/backend-path-prefix](#backend-path-prefix) | `string` | `nil` |
+| [appgw.ingress.kubernetes.io/ssl-redirect](#ssl-redirect) | `bool` | `false` |  |
+| [appgw.ingress.kubernetes.io/connection-draining](#connection-draining) | `bool` | `false` |
+| [appgw.ingress.kubernetes.io/connection-draining-timeout](#connection-draining) | `int32` (seconds) | `30` |
```

Comment:
Should we mention that `default` is 30? Just to be explicit.

Reply:
The column heading is `Default Value`. Should we default in the cell as well ?

Reply:
Missed that . We are good !! 👍